multicluster/test/e2e/service_test.go

```diff
@@ -155,8 +160,67 @@ func (data *MCTestData) deployServiceExport(clusterName string) error {
 func (data *MCTestData) deleteServiceExport(clusterName string) error {
 	rc, _, stderr, err := provider.RunCommandOnNode(clusterName, fmt.Sprintf("kubectl delete -f %s", serviceExportYML))
 	if err != nil || rc != 0 || stderr != "" {
-		return fmt.Errorf("error when deleting the ServiceExport: %v, stderr: %v", err, stderr)
+		return fmt.Errorf("error when deleting the ServiceExport: %v, stderr: %s", err, stderr)
 	}
 
 	return nil
 }
+
+// getNodeName will pick up a Node randomly from Node list
+func getNodeName(nodeName string) (string, error) {
+	rc, output, stderr, err := provider.RunCommandOnNode(nodeName, "kubectl get node -o custom-columns=:metadata.name --no-headers")
+	if err != nil || rc != 0 || stderr != "" {
+		return "", fmt.Errorf("error when getting Node list: %v, stderr: %s", err, stderr)
+	}
+	nodes := strings.Split(output, "\n")
+	source := rand.NewSource(time.Now().UnixNano())
+	random := rand.New(source) // #nosec G404: for test only
+	return nodes[random.Intn(3)], nil
+}
+
+// annotateGatewayNode adds an annotation to assign it as Gateway Node.
+func (data *MCTestData) annotateGatewayNode(clusterName string, nodeName string) error {
+	rc, _, stderr, err := provider.RunCommandOnNode(clusterName, fmt.Sprintf("kubectl annotate node %s multicluster.antrea.io/gateway=true", nodeName))
+	if err != nil || rc != 0 || stderr != "" {
+		return fmt.Errorf("error when annotate the Node %s: %s, stderr: %s", nodeName, err, stderr)
+	}
+	log.Infof("The Node %s is annotated as Gateway in cluster %s", nodeName, clusterName)
+	return nil
+}
+
+func (data *MCTestData) deleteAnnotation(clusterName string, nodeName string) error {
```

Comment:
this method name is too generic, hard to understand it's only to delete `multicluster.antrea.io/gateway` annotation only, maybe name this and above methods `setGatewayNode` and `UnsetGatewayNode`